datamining/views.py

```diff
@@ -42,8 +42,9 @@ def show_members_by_date(request):
     today = datetime.date.today().strftime("%Y-%m-%d")
 
     for member in found_members:
-        if len(member.get_email()) > 0:
-            member_mails.append(member.get_email())
+        email = member.get_email()
+        if email is not None and email:
```

Comment:
A bit pedantic, but None values are falsy anyway, so this is the same as 
`if email`

Reply:
fix